src/Stryker.Core/Stryker.Core/MutantFilters/FilePatternMutantFilter.cs

```diff
@@ -35,15 +35,15 @@ bool IsMutantIncluded(Mutant mutant)
                 }
 
                 // Check if the mutant is excluded.
-                if (_excludePattern.Any(MatchesPattern))
-                {
-                    return false;
-                }
-
-                return true;
+                return !_excludePattern.Any(MatchesPattern);
 
                 bool MatchesPattern(FilePattern pattern)
                 {
+                    // if we do not have the original node, we cannot exclude the mutation according to its location
+                    if (mutant.Mutation.OriginalNode == null)
```

Comment:
In which case would we not have the original node at this point?

Reply:
during unit tests. Curent version explicitly disable filters to avoid this issue. Requiring to have a specific dependency injection just to disable filters for test, which does not seem clean to me.

Better alternative would be to ensure Mutant for test do have an original node.
Alternatively, we could remove the default value for FilePatternMatcher (**.* I think), since it is a systematic passthrough